### PR TITLE
refactor use charsLength instead of chars.length

### PR DESCRIPTION
**Pull request by wongterrencew**
Trivial change, but figured maybe this was accidentally let through :p

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ All notable changes to this project will be documented in this file. If a contri
 
 - Add react-native `ImageBackground` alias (see [#1028](https://github.com/styled-components/styled-components/pull/1028))
 
+- Refactor variable in generateAlphabeticName.js
```
Comment (philpl) [inline CHANGELOG.md]: Can you add the `(...)` note with a link to this PR please? :)

**Comment by k15a** [comment]
I think we did this to cache the property access because it's done pretty often. Don't know if the engines do this optimization from default :)

**Comment by philpl** [comment]
@k15a Yes, I've optimised it and overlooked the residual call there :wink: The engines don't do this optimisation automatically, since the `.length` i.e. the array might be changed by side effects at any time.

**Comment by k15a** [comment]
My bad, I thought this PR changes this back to chars.length but that's not the case.

File: CHANGELOG.md (hunk 1 above)
Comment (wongterrencew) [inline CHANGELOG.md]: 👌  done

**Pull request merged**